packages/perseus/src/widgets/__tests__/radio/choice_test.jsx

```diff
@@ -133,30 +99,54 @@ describe("choice", () => {
         expect(screen.getByText("Correct (selected)")).toBeVisible();
     });
 
-    it("shows incorrect when in review mode", () => {
+    it("shows incorrect selected when in review mode", () => {
         // Arrange / Act
         renderChoice({
-            checked: false,
+            checked: true,
             correct: false,
             showCorrectness: true,
             reviewMode: true,
         });
 
         // Assert
-        expect(screen.getByText("Incorrect")).toBeVisible();
+        expect(screen.getByText("Incorrect (selected)")).toBeVisible();
     });
+});
 
-    it("shows incorrect selected when in review mode", () => {
+describe("choice button", () => {
```

Comment:
nit: it's already pretty clear that you want to test the button vs the checkbox. I just think since it is pretty atypical to have two elements controlling one control that it would be worth to have a comment explaining that this is meant to test 1/2 of those elements and the other tests test 2/2.

Reply:
Added these comments, but let me know if they are not clear enough. Thanks for pointing this out!